keysign/discover.py

```diff
@@ -39,10 +39,12 @@ def start(self):
             key_data = yield threads.deferToThread(self.discovery.find_key, self.userdata)
             log.debug("Received key successfully")
         except ValueError:
+            exc_type, exc_value = exc_info()[:2]
+            log.debug('Handling %s exception with message "%s"', exc_type.__name__, exc_value)
```

Comment:
what? No.
`log.debug` takes a parameter for the exception.

Reply:
Ahh, I'm sorry. let me correct it.

Reply:
@muelli  do we also need Traceback?

Reply:
just call `log.debug("foo", exc_info=e)` or so.